databases/business-sync/fragment-shoebox-snippet.js

```diff
@@ -0,0 +1,40 @@
+{
+  channels: getDocSyncChannels(doc, oldDoc),
+  typeFilter: function(doc, oldDoc) {
+    return createBusinessEntityRegex('shoeboxSnippet\\.[A-Za-z0-9_-]+\\.[A-Za-z0-9_-]+').test(doc._id);
```

Comment:
I feel that `shoeboxSnippet` is a bit wordy and maybe anachronistic (i like using that word). Is it your preferred noun? Should we brainstorm for other nouns? `dataSnippet`, `rawData`, `sourceData` ...? :brain: ⛈